### ISSUE TITLE
loss

### ISSUE THREAD

**Issue by zhou431496** [cvg/LightGlue#17]
Hi, great work, how can I calculate the difference of keypoints of two images and backpropagate it as a loss

**Comment by Phil26AT** [comment]
We will release the loss function of LightGlue together with the training code (we used an NLL-loss with ground-truth correspondences, as discussed in the paper).
